app/src/ui/dialog/dialog.tsx

```diff
@@ -526,19 +526,20 @@ export class Dialog extends React.Component<IDialogProps, IDialogState> {
     if (!e) {
       if (this.dialogElement) {
         this.dialogElement.removeEventListener('cancel', this.onDialogCancel)
-        this.dialogElement.removeEventListener('keydown', this.onKeyDown)
       }
     } else {
       e.addEventListener('cancel', this.onDialogCancel)
-      e.addEventListener('keydown', this.onKeyDown)
     }
 
     this.dialogElement = e
   }
 
-  private onKeyDown = (event: KeyboardEvent) => {
+  private onKeyDown = (event: React.KeyboardEvent) => {
     const shortcutKey = __DARWIN__ ? event.metaKey : event.ctrlKey
     if ((shortcutKey && event.key === 'w') || event.key === 'Escape') {
+      if (event.defaultPrevented) {
```

Comment:
Could we move this to the top of the method such that we return early if the event has already been handled? While it doesn't affect the logic at all it would follow the pattern we use elsewhere in the app

https://github.com/desktop/desktop/blob/21529147666b7beb6102594c3167ccea024b9c9c/app/src/ui/app.tsx#L901-L904

https://github.com/desktop/desktop/blob/21529147666b7beb6102594c3167ccea024b9c9c/app/src/ui/repository.tsx#L474-L477

etc

Reply:
Sorry I didn't catch that sooner, already done in 69dbe27.